docs/support-issue-template.md

```diff
@@ -0,0 +1,22 @@
+# [Issue Title]
```

Comment:
We need to add the following at the beginning:
```
---
id: support_issue_template
title: 
---
```
and then we need to add the id to the sidebar.js in the relevant section so that we can view the page from the sidebar :)